docs/topics/db/models.txt

```diff
@@ -1415,3 +1415,47 @@ any model field in any ancestor model.
     :doc:`The Models Reference </ref/models/index>`
         Covers all the model related APIs including model fields, related
         objects, and ``QuerySet``.
+
+Organizing models
+=================
+
+The :djadmin:`manage.py startapp <startapp>` script creates a package structure
+for an application that includes a file called ``models.py``, but you may
+also define multiple files containing models, contained in a module. For example,
+you can remove the ``models.py`` and create a ``myproject/myapp/models/`` with
+two files (besides an ``__init__.py``), ``person_models.py`` and ``robot_models.py``:
+
+``person_models.py``::
```

Comment:
You can use `.. snippet::` with `:filename:` instead, however, I don't think it's important to show these two files. Defining models is already covered.
As Simon suggested, I'd drop the `_models` suffix.